packages/influx-utilities/package.json

```
@@ -0,0 +1,65 @@
+{
+	"name": "@skyra/influx-utilities",
+	"version": "0.0.1",
+	"description": "A tiny opinionated abstraction layer for InfluxDB for interacting with a single organization",
+	"author": "@skyra",
+	"license": "Apache-2.0",
+	"main": "dist/index.js",
+	"module": "dist/index.mjs",
+	"types": "dist/index.d.ts",
+	"type": "commonjs",
+	"exports": {
+		"import": "./dist/index.mjs",
+		"require": "./dist/index.js",
+		"types": "./dist/index.d.ts"
```

```suggestion
		"types": "./dist/index.d.ts",
		"default": "./dist/index.mjs"
```

- types needs to be on top
- default does the job here as this is ESM only (yes I know other packages should get this change too)
Also see: https://nodejs.org/api/packages.html#conditional-exports and https://github.com/sapphiredev/shapeshift/pull/273#issue-1737643314